app/views/users/profile.html.erb

```diff
@@ -296,6 +298,48 @@
       $("#social-links").toggle();
   });
 
+  $(document).ready(function() {
+    function splitText(text) {
+        var textBreak = textLimit;
+        var first = text.substring(0, textBreak);
+        var second = text.substring(textBreak);
+        var aux = second.substring(0, second.indexOf(" "));
+        var spaceIndex = second.indexOf(" ");
+        second = " " + second.substring(spaceIndex + 1);
+        first = first.substring(0, textBreak) + aux;
+        var bothTextes = [first, second];
+        return bothTextes;
```

Comment:
please add a newline before return statement, this contributes towards more readable code.

Reply:
you can just return the array as `return [first, second]`

Reply:
That's much better @gauravano :tada: It slipped out of my mind completely. Thank you for suggesting it.